plugin/pkg/admission/persistentvolume/label/admission.go

```diff
@@ -131,7 +137,11 @@ func (l *persistentVolumeLabel) getEBSVolumes() (aws.Volumes, error) {
 	defer l.mutex.Unlock()
 
 	if l.ebsVolumes == nil {
-		cloudProvider, err := cloudprovider.GetCloudProvider("aws", nil)
+		var cloudConfigReader io.Reader
+		if len(l.cloudConfig) > 0 {
+			cloudConfigReader = bytes.NewReader(l.cloudConfig)
+		}
+		cloudProvider, err := cloudprovider.GetCloudProvider("aws", cloudConfigReader)
```

Comment:
okay, that sounds fair. However, federation api sever is in this weird place where it accepts same set of arguments as api server but sometimes doesn't do anything with them. 

I do not want to initialize cloudprovider in federation server as part of this change because I don't know what their expectations are. If we are going to directly pass cloudprovider then I will pass `nil` from federation apiserver until they decide to initialize cloudprovider in their code and  it becomes available. Does it sound good?